docs/import-images.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Import images"
+---
+
+A container image represents binary data that encapsulates an application and all its software dependencies. Container images are self-contained, executable software bundles that can run standalone and make well-defined assumptions about their runtime environment. Container images for an application are typically created and then pushed to a registry. K3s, as a CNCF-certified Kubernetes distribution, relies on containerd to pull the required images for Pods to run correctly.
```

Comment:
Do we need to start out with a paragraph describing what images are and how they are used?

Reply:
I thought it might be useful for newbies but I guess everyone should understand images when they come to these docs